websites/C/Claude/presence.ts

```diff
@@ -0,0 +1,90 @@
+import { franc } from "franc-min";
+
+const presence = new Presence({
+		clientId: "1326893434073776181",
+	}),
+	getStrings = async () => {
+		return presence.getStrings(
+			{
+				talkingWithAI: "claude.talkingWithAI",
+				aiResponding: "claude.aiResponding",
+				conversationStats: "claude.conversationStats",
+				startNewConversation: "claude.startNewConversation",
+				thinkingOfPrompt: "claude.thinkingOfPrompt",
+			},
+			await presence.getSetting<string>("lang").catch(() => "en")
+		);
+	},
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let oldLang: string = null,
+	strings: Awaited<ReturnType<typeof getStrings>>;
+
+const enum Assets {
+	Logo = "https://i.imgur.com/SqUGiUx.png",
+	Talking = "https://cdn.rcd.gg/PreMiD/websites/C/ChatGPT/assets/0.png",
```

Comment:
Rather than this, please re-upload the asset to a service like Imgur and use that link.